theano/compile/mode.py

```diff
@@ -18,35 +18,6 @@
 _logger = logging.getLogger('theano.compile.mode')
 
 
-def check_equal(x, y):
```

Comment:
This cause a failure in our test as now, the numpy import isn't used anymore in that file. Can you remove the numpy import from that file?

/home/travis/build/Theano/Theano/theano/compile/mode.py:8:1: F401 'numpy' imported but unused